ivw_integration.install

```diff
@@ -27,3 +27,87 @@ function ivw_integration_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Add missing field columns to field tables.
+ *
+ * @param string $field_type
+ *   The field types id.
+ *
+ * @return array
+ *   Gathered messages.
+ */
+function ivw_integration_add_missing_field_columns($field_type) {
+  $entity_type_manager = \Drupal::entityTypeManager();
+  $database_schema = \Drupal::database()->schema();
+
+  // Get all of the field storage entities of type ivw_integration_settings.
+  $field_storage_configs = $entity_type_manager
+    ->getStorage('field_storage_config')
+    ->loadByProperties(['type' => $field_type]);
+
+  foreach ($field_storage_configs as $key => $field_storage) {
+    // The name of the field.
+    $field_name = $field_storage->getName();
+    $field_schema = $field_storage->getSchema();
+
+    // The entity type to which this field is attached.
+    $target_entity_type_id = $field_storage->getTargetEntityTypeId();
+    $target_entity_type = $entity_type_manager->getDefinition($target_entity_type_id);
+    $target_entity_type_storage = $entity_type_manager->getStorage($target_entity_type_id);
+
+    // Table mapping is used to generate the table names of the found fields.
+    $table_mapping = $target_entity_type_storage->getTableMapping();
+
+    // Get the table name, that needs to be updated.
+    $field_table_names = [$table_mapping->getDedicatedDataTableName($field_storage)];
+
+    // If the entity is revisionable, also update the revision table.
+    if ($target_entity_type->isRevisionable()) {
+      $field_table_names[] = $table_mapping->getDedicatedRevisionTableName(
+        $field_storage
+      );
+    }
+
+    foreach ($field_table_names as $field_table_name) {
+      if (!$database_schema->tableExists($field_table_name)) {
+        continue;
+      }
+      foreach ($field_schema['columns'] as $column => $definition) {
+        $field_column_names = $table_mapping->getColumnNames($field_name);
+
+        // The actual column name in the database for this column in this field and entity.
+        $column_name = $field_column_names[$column];
+        if (!$database_schema->fieldExists(
+            $field_table_name,
+            $column_name
+          )
+        ) {
+          $messages[] = t(
```

Comment:
$messages should be initialized at beginning as empty array, so that array is always returned from function. In case there is no IVW field -> we get error: ```implode(): Invalid arguments passed ivw_integration.install:112```.